spec/requests/api/milestones_spec.rb

```diff
@@ -86,6 +100,14 @@
 
       json_response['state'].should == 'closed'
     end
+
+    it "should update a project milestone by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.